### ISSUE TITLE
Game DDoS Protection: as-a-service mode

### ISSUE THREAD

**Issue by jslocinski** [ovh/infrastructure-roadmap#85]
As a game hosting company I would like to use Game DDoS Protection for any backend (incl. VPS, PCI instances and other bare-metal ranges) to best fit protection with a server range.

**Comment by allensw00** [comment]
Agree, especially the newest game range is basically advanced + game firewall (just look at the CPU)

**Comment by alexevladgabriel** [comment]
@jslocinski Is there new updates regarding this or steps of current architecture progress to allow this, in this year?
